### DISCUSSION TITLE
DataAnnotations e campos Boolean

### DISCUSSION THREAD

**Discussion by zetunejacques** [balta-io/7194#11] [Perguntas e Respostas] [answered]
Estou tentando decorar um campo do tipo boolean para valiudação.
Tentei de duas formas: usando [RegularExpression] e criando um a validação Custom mas não consegui.
Alguém poderia indicar qual a melhor prática para se validar campos booleanos?

**Comment by andrebaltieri** [comment, marked as answer]
@zetunejacques qual custom você escreveu? Consegue compartilhar o código aqui?
Sobre o Regex, ele só funciona sob strings!
